### PR TITLE
NUP-2423: Fix tests

### PR DESCRIPTION
**Pull request by marionleborgne**
Fixes https://github.com/numenta/nupic/issues/3571

### CODE REVIEW

File: tests/unit/nupic/docs/examples_test.py [hunk 1]
```diff
@@ -91,8 +91,6 @@ def testExamplesDirExists(self):
     self.assertTrue(os.path.exists(ExamplesTest.examplesDir), failMsg)
 
 
-  @unittest.skip("Skip test until we figure out why we get different "
-                 "results with OPF, Network and Algorithm APIs.")
```
Comment (rhyolight) [inline tests/unit/nupic/docs/examples_test.py]: Did you intentionally remove this skip?

Comment (marionleborgne) [inline tests/unit/nupic/docs/examples_test.py]: Yes. There was a simple fix.

Comment (marionleborgne) [inline tests/unit/nupic/docs/examples_test.py]: https://github.com/numenta/nupic/pull/3570/commits/41afe3b94ede5283b4b612372a6145135c2b2734

**Comment by rhyolight** [comment]
Looks like this fixed builds. 👍

**Pull request merged**